experiment_pages/experiment/data_collection_ui.py

```diff
@@ -127,6 +127,19 @@ def change_selected_value(self, values):
             self.open_auto_increment_changer()
         AudioManager.play(filepath="shared/sounds/rfid_success.wav") #play succsess sound
 
+        # Potential modify to ensure it interacts with the DataCollectionDatabase properly
+        # if "None" in old_measurements:
+        #     # Save new entry using DataCollectionDatabase
+        #     self.data_database.set_data_for_entry(
+        #         (str(date.today()), animal_id) + tuple(new_values)
+        #     )
+        # else:
+        #     # Update existing entry
+        #     self.data_database.set_data_for_entry(
+        #         (str(date.today()), animal_id) + tuple(new_values)
+        #     )
+
+
```

Comment:
I see how this can ensure the changes are saved, but is it needed for every increment?

Reply:
1. Be sure to uncomment code so the automatic workflows can check on it, and so you can validate functionality
2. Is this code intended to be a part of the method 'def change_selected_value(...)? If so, there is already ```if("None" in old_measurements):
            self.database.add_data_entry(date.today(), animal_id, new_values)
        else:
            self.database.change_data_entry(date.today(), animal_id, new_values)``` on lines 119-122, which appears to do the same thing.